cc_licenses/cc_licenses/settings.py

```diff
@@ -0,0 +1,113 @@
+"""
+Django settings for cc_licenses project.
+
+Generated by 'django-admin startproject' using Django 3.0.7.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.0/ref/settings/
+"""
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "g*l49b_4r6dmvh9&=3u)qfv@s!%ln6jgvi)_wgstlmn#_egzpn"
```

Comment:
we usually don't commit the `SECRET_KEY` for our Django projects just so nobody deploys it accidentally with a key that's public. cc @aldenstpage

Reply:
Thank you for catching this. Going to follow up with you about this.

Reply:
In this case it's OK to load it from an environment variable without a default so it fails to start if someone has forgotten to set it. We do the same with our own Django projects. https://github.com/creativecommons/cccatalog-api/blob/master/cccatalog-api/cccatalog/settings.py#L26

The Django secret key doesn't need to be shared between us, we can generate our own when we deploy it.

Reply:
@aldenstpage and @kgodey, pushed changes addressing our conversation above.